src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/domain/repository/AlnScreenerRepository.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.supportadditionalneedsapi.domain.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.supportadditionalneedsapi.domain.entity.ALNScreenerEntity
+import java.util.UUID
+
+@Repository
+interface AlnScreenerRepository : JpaRepository<ALNScreenerEntity, UUID> {
+
+  fun getAllByPrisonNumber(prisonNumber: String): List<ALNScreenerEntity>
```

Comment:
Minor, and probs me showing (again!) how little I know about JPA etc 😬  ... but what is the difference between `getAllByPrisonNumber` and `findByPrisonNumber` ?
They both appear to return a list (albeit one is mutable) ....

As a consumer of this repository class, should I know which method to be calling? 🤔